src/features/pass-survey/model/AnswerValidator.ts

```diff
@@ -106,6 +106,16 @@ function AnswerValidator(params?: AnswerValidatorArgs): IAnswerValidator {
 
           return true;
         }
+        case 'ParagraphText': {
+          const numberOfCharacter = `${currentAnswer?.answer}`.length;
```

Comment:
Add `answer` cast to `ParagraphTextResponse`
```suggestion
          const answer = currentAnswer?.answer as ParagraphTextResponse;
          const limit = currentPipelineItem.payload.maxLength;

          if (limit < answer.length) {
            return false;
          }
```